bin/hdfcoinc/pycbc_page_foundmissed

```diff
@@ -243,4 +253,4 @@ if ('.html' in args.output_file):
 pycbc.results.save_fig_with_metadata\
```

Comment:
Could you also PEP-ify this call? I would prepare a `title` variable before the call so you can just pass `title=title` and semplify the arg list.

Reply:
Done (I hadn't actually scrolled that far in the tidy up!)